packages/tailwindcss-language-service/src/util/rewriting/replacements.ts

```diff
@@ -58,9 +77,20 @@ export function replaceCssVars(str: string, replace: CssVarReplacer): string {
           str = str.slice(0, i) + replacement + str.slice(j + 1)
```

Comment:
<img width="270" alt="Screenshot 2025-02-05 at 13 28 32" src="https://github.com/user-attachments/assets/f1364f71-4acc-42fd-b7a1-ca1bcf983dc0" />

LGTM but we should handle `\` here 🙈

Reply:
Fixed that too